Tests/Parser/Client/fixtures/mobile_app.yml

```diff
@@ -2105,3 +2105,21 @@
     type: mobile app
     name: Perplexity
     version: ""
+-
+  user_agent: Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/14G60 QQ/103.3 TIM/3.3.5.545 V1_IPH_SQ_8.4.8_335_TIM_D Pixel/750 MiniAppEnable SimpleUISwitch/1 QQTheme/1015712 Core/WKWebView Device/Apple(iPhone 6) NetType/4G QBWebViewType/1 WKType/1
+  client:
+    type: mobile app
+    name: TIM
+    version: 3.3.5.545
+-
+  user_agent: Mozilla/5.0 (iPad; CPU OS 17_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 QQ/7.2.0.0 TencentDocs/3.1.7 Pixel/1668 Core/WKWebView Device/Apple(iPad Pro (11-inch) (4th generation)) NetType/undefined _tdocFlag/2 exkb/1 QBWebViewType/1 WKType/1
+  client:
+    type: mobile app
+    name: Tencent Docs
+    version: 3.1.7
+-
+  user_agent: Mozilla/5.0 (iPhone; CPU iPhone OS 17_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 QQ/9.0.85.612 V1_IPH_SQ_9.0.85_1_APP_A Pixel/1179 MiniAppEnable SimpleUISwitch/0 StudyMode/0 CurrentMode/0 CurrentFontScale/1.000000 QQTheme/1000 AppId/537235392 Core/WKWebView Device/Apple(iPhone X) NetType/WIFI QBWebViewType/1 WKType/1
+  client:
+    type: mobile app
+    name: QQ
+    version: 9.0.85.612
```

Comment:
https://browser.qq.com/
https://browser.qq.com/mobile/article_detail/3/1/365

indeed, we should leave QQ in applications, this is both the browser and the document office?

Reply:
> indeed, we should leave QQ in applications, this is both the browser and the document office?

Probably it has some built in viewer. the browser uses only `MQQBrowser` (just tested) while the app uses `QQ`.